source/index.ts

```diff
@@ -2,7 +2,7 @@ import {EventEmitter} from 'events';
 import {Socket} from 'net';
 import {ClientRequest, IncomingMessage} from 'http';
 // @ts-ignore
-import deferToConnect from 'defer-to-connect';
+import deferToConnect = require('defer-to-connect');
```

Comment:
why this?

Reply:
It's not a TS module. @sindresorhus has explained it somewhere but I can't find it now.

Reply:
It's also explained here: https://github.com/DefinitelyTyped/DefinitelyTyped/#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports

Reply:
Feel free to pick up https://github.com/szmarczak/defer-to-connect/issues/2